src/arrayMethodSort.js

```diff
@@ -4,8 +4,21 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
+  ) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
+          const word = this[i];
+
+          this[i] = this[j];
+          this[j] = word;
+        }
+      }
+    }
+
+    return this;
```

Comment:
The current implementation uses a bubble sort algorithm, which has O(n^2) time complexity. For larger arrays, this might not be efficient. If performance is a concern, consider researching and implementing more efficient sorting algorithms like quicksort or mergesort.